docs/Makefile

```diff
@@ -3,7 +3,6 @@
 IMAGE = grafana/docs-base:latest
 
 docs:
-	docker pull ${IMAGE}
```

Comment:
no idea how others test with new images, but I have to always remove this so that it does not overwrite my locally built image